lib/settings_base.py

```diff
@@ -525,6 +526,7 @@ def JINJA_CONFIG():
             'css/impala/compat.less',
             'css/impala/localizers.less',
             'css/impala/fxa-migration.less',
+            'css/restyle.css',
```

Comment:
I just stuck with a plain old css file since it was mostly a straight up port from the Stylish theme.